### ISSUE TITLE
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "xxd": executable file not found in $PATH: unknown

### ISSUE THREAD

**Issue by fabiooshiro** [gbarros/unshackle-cartesi#1]
Hi @gbarros, when I try to run the command below I got this error:
```
DAPP_NAME=echo-rust docker compose --env-file ../env.rollux_testnet -f  ../deploy-testnet.yml up
[+] Running 3/0
 ✔ Container rollups-examples-tim-machine-1             Recreated                                                                                            0.0s 
 ✔ Container rollups-examples-tim-authority-deployer-1  Created                                                                                              0.0s 
 ✔ Container rollups-examples-tim-dapp-deployer-1       Recreated                                                                                            0.0s 
Attaching to authority-deployer-1, dapp-deployer-1, machine-1
authority-deployer-1  | rollux_testnet 123
authority-deployer-1  | Nothing to compile
authority-deployer-1  | No need to generate any newer typings.
authority-deployer-1  | reusing "Bitmask" at 0xF5B2d8c81cDE4D6238bBf20D3D77DB37df13f735
authority-deployer-1  | reusing "CartesiMathV2" at 0xB634F716BEd5Dd5A2b9a91C92474C499e50Cb27D
authority-deployer-1  | reusing "MerkleV2" at 0x33436035441927Df1a73FE3AAC5906854632e53d
authority-deployer-1  | reusing "UnrolledCordic" at 0x3F8FdcD1B0F421D817BF58C96b7C91B98100B450
authority-deployer-1  | reusing "InputBox" at 0x59b22D57D4f067708AB0c00552767405926dc768
authority-deployer-1  | reusing "EtherPortal" at 0xFfdbe43d4c855BF7e0f105c400A50857f53AB044
authority-deployer-1  | reusing "ERC20Portal" at 0x9C21AEb2093C32DDbC53eEF24B873BDCd1aDa1DB
authority-deployer-1  | reusing "ERC721Portal" at 0x237F8DD094C0e47f4236f12b4Fa01d6Dae89fb87
authority-deployer-1  | reusing "ERC1155SinglePortal" at 0x7CFB0193Ca87eB6e48056885E026552c3A941FC4
authority-deployer-1  | reusing "ERC1155BatchPortal" at 0xedB53860A6B52bbb7561Ad596416ee9965B055Aa
authority-deployer-1  | reusing "DAppAddressRelay" at 0xF5DE34d6BbC0446E2a45719E718efEbaaE179daE
authority-deployer-1  | reusing "CartesiDAppFactory" at 0x7122cd1221C20892234186facfE8615e6743Ab02
authority-deployer-1  | reusing "Authority" at 0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785
authority-deployer-1  | reusing "History" at 0x4FF8BD9122b7D91d56Dd5c88FE6891Fb3c0b5281
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "xxd": executable file not found in $PATH: unknown
Gracefully stopping... (press Ctrl+C again to force)
```

**Comment by fabiooshiro** [comment]
@sandhilt and I changed a few lines and the error now is:
<img width="910" alt="image" src="https://github.com/gbarros/unshackle-cartesi/assets/218768/d4241bd6-7119-450f-b6c5-0b9ab02389ee">
```shell
dapp-deployer-1       | Deploying DApp "echo-python" on network "rollux_testnet" using consensus address "0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785"...
dapp-deployer-1       | connecting to https://rpc-tanenbaum.rollux.com
dapp-deployer-1       | unsupported network 57000
dapp-deployer-1 exited with code 1
```

**Comment by fabiooshiro** [comment]
We did everything again and now it's working:
```shell
DAPP_NAME=echo-python docker compose --env-file ../env.rollux_testnet -f  ../deploy-testnet.yml up
```
The end of output is:
```shell
authority-deployer-1  | rollux_testnet 123
authority-deployer-1  | Nothing to compile
authority-deployer-1  | No need to generate any newer typings.
authority-deployer-1  | reusing "Bitmask" at 0xF5B2d8c81cDE4D6238bBf20D3D77DB37df13f735
authority-deployer-1  | reusing "CartesiMathV2" at 0xB634F716BEd5Dd5A2b9a91C92474C499e50Cb27D
authority-deployer-1  | reusing "MerkleV2" at 0x33436035441927Df1a73FE3AAC5906854632e53d
authority-deployer-1  | reusing "UnrolledCordic" at 0x3F8FdcD1B0F421D817BF58C96b7C91B98100B450
authority-deployer-1  | reusing "InputBox" at 0x59b22D57D4f067708AB0c00552767405926dc768
authority-deployer-1  | reusing "EtherPortal" at 0xFfdbe43d4c855BF7e0f105c400A50857f53AB044
authority-deployer-1  | reusing "ERC20Portal" at 0x9C21AEb2093C32DDbC53eEF24B873BDCd1aDa1DB
authority-deployer-1  | reusing "ERC721Portal" at 0x237F8DD094C0e47f4236f12b4Fa01d6Dae89fb87
authority-deployer-1  | reusing "ERC1155SinglePortal" at 0x7CFB0193Ca87eB6e48056885E026552c3A941FC4
authority-deployer-1  | reusing "ERC1155BatchPortal" at 0xedB53860A6B52bbb7561Ad596416ee9965B055Aa
authority-deployer-1  | reusing "DAppAddressRelay" at 0xF5DE34d6BbC0446E2a45719E718efEbaaE179daE
authority-deployer-1  | reusing "CartesiDAppFactory" at 0x7122cd1221C20892234186facfE8615e6743Ab02
authority-deployer-1  | reusing "Authority" at 0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785
authority-deployer-1  | reusing "History" at 0x4FF8BD9122b7D91d56Dd5c88FE6891Fb3c0b5281
authority-deployer-1  | npm notice 
authority-deployer-1  | npm notice New major version of npm available! 8.15.0 -> 10.4.0
authority-deployer-1  | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.4.0>
authority-deployer-1  | npm notice Run `npm install -g npm@10.4.0` to update!
authority-deployer-1  | npm notice 
authority-deployer-1 exited with code 0
dapp-deployer-1       | Deploying DApp "echo-python" on network "rollux_testnet" using consensus address "0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785"...
dapp-deployer-1       | connecting to https://rpc-tanenbaum.rollux.com
dapp-deployer-1       | network not supported by default, trying local file
dapp-deployer-1       | network not supported by default, trying local file
dapp-deployer-1       | using account "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
dapp-deployer-1       | transaction: 0x6166ded54984b54ad493c97c3085b9926a296b480241a149385c68c9d962b737
dapp-deployer-1       | waiting for confirmation...
dapp-deployer-1       | application: 0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C
dapp-deployer-1       | writing application address to /deployments/rollux_testnet/echo-python.json
dapp-deployer-1 exited with code 0
```
We've kept the changes in the deploy-testnet.yml file

**Comment by fabiooshiro** [comment]
To help others here is the full content of deploy-testnet.yml:
```
version: "3.9"

services:
  machine:
    # image: cartesi/dapp:${DAPP_NAME:?undefined DAPP_NAME}-devel-machine
    image: cartesi/dapp:${DAPP_NAME:?undefined DAPP_NAME}-devel-server
    command: apt update && apt install xxd && xxd -c 256 -p latest/hash; sleep 3
    volumes:
      - machine:/var/opt/cartesi/machine-snapshots

  authority-deployer:
    image: gbbabarros/rollups-hardhat:1.0.1
    command:
      [
        "deploy",
        "--network",
        "${NETWORK:?undefined NETWORK}",
        "--export",
        "/home/node/rollups.json",
      ]
    volumes:
      - ./deployments/${NETWORK:?undefined NETWORK}:/home/node
    environment:
      - MNEMONIC=${MNEMONIC:?undefined MNEMONIC}
      - RPC_URL=${RPC_URL:?undefined RPC_URL}
      - NETWORK=${NETWORK:?undefined NETWORK}
      - CHAIN_ID=57000

  dapp-deployer:
    build: ./build/testnet-dapp-deployer
    depends_on:
      machine:
        condition: service_started
      authority-deployer:
        condition: service_completed_successfully
    volumes:
      - machine:/var/opt/cartesi/machine-snapshots:ro
      - ./deployments:/deployments
      - ./build:/build
    environment:
      - MNEMONIC=${MNEMONIC:?undefined MNEMONIC}
      - NETWORK=${NETWORK:?undefined NETWORK}
      - RPC_URL=${RPC_URL:?undefined RPC_URL}
      - DAPP_NAME=${DAPP_NAME:?undefined DAPP_NAME}
      - CHAIN_ID=57000

volumes:
  machine: {}
```
